R/app_server.R

```diff
@@ -53,7 +59,7 @@ server = function(input, output, session) {
   # get table file path
   table_file = shiny::reactive({
     shiny::req(input$cluster_id)
-    table_file = glue::glue("www/data/wcdemo/scanner_output/{input$cluster_id}/{input$table_type}") # nolint
+    table_file = glue::glue("inst/app/www/data/scanner_output/{input$cluster_id}/{input$table_type}") # nolint
```

Comment:
If I setwd() outside of the package root and `run_app()`, an error is displayed because the file can't be found.
For portability, the inst/ directory shouldn't be mentioned in the filepath. Can you replace with system.file("app", "www", ..., input$cluster_id, input$table_type, package = "tfpbrowser"), please.